### ISSUE TITLE
The nerf on moral is too big

### ISSUE THREAD

**Issue by Nipodemos** [Beefster09/ONI-Rebalancing-Mod#1]
Hey!
I would like to tak to you about the moral nerfs you made on this mod.
Some guy on Steam commented that which I agree with:
```
The morale change is just ridiculous. 
You'll need to consider that there are only 3 quatly 3 food in the game (berry sludge, BBQ, fish),
everything better requires natural gas so its late game only.
Combined with great hall and nature reserve nerf, we are talking about loosing 2+3+6 =11 morale in the mid game,
I just can't see how this would be fair.
```

we losing 3 moral on great hall
6 moral on barbecue
and 2 moral on park

that's just too much. Could you consider revisit those balances and see how they could be improved?

my suggestions would be:
Since now is harder to have a great hall, make it +4 morale (instead of 3)
Since you already nerfed how much morale a food quality meal provides, at least preserve the food quality of the bbq and fish

the park is a very hard room to make, make pips plant 4 plants and put the room in a strategical place so dupes has to pass on it it's a hard task on its own. By making such hardwork provide only +2 moral makes impratical

**Comment by Beefster09** [comment]
I appreciate you reaching out. I obviously haven't been actively working on this mod for a while, but it's good to hear that people have taken it for a spin and it's good to have some feedback.

I have played around on a spaced out colony with the mod and have noticed that morale is very tight, especially when doing space missions. Dupe training became much more constrained than before, which I found to be rather interesting, but I can see where you're coming from in that I really didn't have options to go crazy with recreational buildings until late game.

I won't promise to make the exact changes you're suggesting, but I value your feedback and think you make some good points on issues worth addressing.
